yt/data_objects/construction_data_containers.py

```diff
@@ -1542,7 +1542,7 @@ def get_data(self, fields=None, sample_type="face", no_ghost=False):
             mylog.info("Extracting (sampling: %s)" % (fields,))
         verts = []
         samples = []
-        for io_chunk in parallel_objects(self.data_source.chunks([], "io")):
+        for _io_chunk in parallel_objects(self.data_source.chunks([], "io")):
```

Comment:
weird that this is not used. Is this a bug ?

Reply:
Would it make sense to use `io_chunk` inside the loop in lieu of `self.data_source` then ?